### PR TITLE
media-video/plasmatube: new package, add 23.08.49.9999, 9999

### PR DESCRIPTION
**Pull request by CyberTailor**

### CODE REVIEW

File: media-video/plasmatube/plasmatube-23.08.49.9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	>=kde-frameworks/kconfig-${KFMIN}:5
+	>=kde-frameworks/kcoreaddons-${KFMIN}:5
+	>=kde-frameworks/ki18n-${KFMIN}:5
+	>=kde-frameworks/kirigami-${KFMIN}:5
+	media-video/mpv:=[libmpv]
+"
+RDEPEND="${DEPEND}
+	dev-libs/kirigami-addons:5
+	>=dev-qt/qtgraphicaleffects-${QTMIN}:5
+	>=dev-qt/qtmultimedia-${QTMIN}:5[qml]
```
Comment (redstrate) [inline media-video/plasmatube/plasmatube-23.08.49.9999.ebuild:34]: Technically not needed, we don't use qtmultimedia ourselves but some parts of kirigami-addons might.

**Comment by redstrate** [comment]
23.08 ebuild looks fine, but the 9999 build will fail since master is now Qt6 and the deps changed.

File: media-video/plasmatube/plasmatube-23.08.49.9999.ebuild (hunk 1 above)
Comment (CyberTailor) [reply media-video/plasmatube/plasmatube-23.08.49.9999.ebuild:34]: "src/contents/ui/videoplayer" imports QtMultimedia

**Comment by CyberTailor** [comment]
AFAIK 9999 is not used directly but needed for "regenerate-files" script

**Comment by a17r** [comment]
9999 is used by whomever wants to use 9999, so if you want to add it, it should also have correct dependencies.

File: media-video/plasmatube/plasmatube-23.08.49.9999.ebuild (hunk 1 above)
Comment (redstrate) [reply media-video/plasmatube/plasmatube-23.08.49.9999.ebuild:34]: You're right, that still applies to 23.08 just not in 9999

**Pull request closed without merging**